crates/context/config/src/client/config.rs

```diff
@@ -0,0 +1,40 @@
+use std::collections::BTreeMap;
+
+use near_crypto::SecretKey;
+use near_primitives::types::AccountId;
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct ContextConfigClientConfig {
+    pub signer: ContextConfigClientSigner,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct ContextConfigClientSigner {
+    #[serde(rename = "use")]
+    pub selected: ContextConfigClientSelectedSigner,
+    pub relayer: ContextConfigClientRelayerSigner,
+    #[serde(rename = "self")]
+    pub local: BTreeMap<String, ContextConfigClientLocalSigner>,
+}
```

Comment:
It would be great to start adding in Rustdocs to everything new we add 🙂

Reply:
There's no time for this, which is why the PR description includes a preview of what this does.